app/(main)/(drawer)/(apply-now)/edit-application/[pageID].tsx

```
@@ -0,0 +1,704 @@
+      setOriginalAttachments(
+        applicationData.attachments as {
+          id: string;
+          imageUrl: string;
+          playUrl: string;
+          appleUrl: string;
+          type: string;
+        }[]
+      );
+      setAnswers(
+        applicationData.answersFromInfluencer.reduce(
+          (acc, curr) => ({ ...acc, [curr.question]: curr.answer }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)